build.fsx

```diff
@@ -70,32 +70,31 @@ let isXamarinPlatform = false //EnvironmentHelper.isMacOS || Environment.OSVersi
 
 
 // Helper active pattern for project types
-let (|Fsproj|Csproj|Vbproj|) (projFileName:string) = 
+let (|Fsproj|Csproj|Vbproj|) (projFileName:string) =
     match projFileName with
     | f when f.EndsWith "fsproj" -> Fsproj
     | f when f.EndsWith "csproj" -> Csproj
     | f when f.EndsWith "vbproj" -> Vbproj
     | _                           -> failwith (sprintf "Project file %s not supported. Unknown project type." projFileName)
 
 
-let activeProjects = 
-    let xamarinFilter f = 
+let activeProjects =
+    let xamarinFilter f =
         if isXamarinPlatform then
             f
         else
             f
             -- "**/OpenTK.Android.csproj"
             -- "**/OpenTK.iOS.csproj"
-            
+
     !! "src/**/*.??proj"
     ++ "tests/**/OpenTK.Tests.fsproj"
-    -- "**/OpenTK.GLWidget.csproj"
```

Comment:
This is the only real change in this file. Trailing spaces were automatically trimmed and I had not noticed until now.